exist-versioning-release.md

```diff
@@ -93,40 +93,121 @@ That version number is also copied into the `META-INF/MANIFEST.MF` file of any J
 
+A valid Apple Developer Certificate to sign Mac DMG is required. If you don't have one you will need to join the [Apple Developer Program](https://developer.apple.com/programs/enroll/) and to create a certificate via XCode. 
+
+##### Github
+
+A Github account and username / password or Github Personal (access tokens)[https://github.com/settings/tokens] with permission to publish Github releases to the eXist-db org. 
+
+##### Sonatype OSS
+
+You will need a Sonatype OSS acount which has been authorized for the eXist-db organisation at Sonatype OSS staging for Maven Central - https://oss.sonatype.org/
+
+
+
+##### Docker 
+
+A DockerHub user with permissions to publish to https://cloud.docker.com/orgs/existdb/ is required. 
+
+
+#### Prepare Release Notes 
+
+* There are various possibilities to create the release notes.
+
+> This section is still work in progress and manual tasks are needed to generate the release notes
+
+The release notes start with an introduction summarizing the most important changes for end-users and developers since the last version was released.
+
+Three sections follow
+
+- a list of all changes and features with links to their PRs
+- a list of all fixes with links the their PRs
+- a list of all dependencies that got updated along with the new version that is now used in existdb
+
+##### gitchangelog (Python) 
+
+* https://pypi.org/project/gitchangelog/
+* configure .gitchangelog.rc in the root of eXist-db repo
+* run `gitchangelog` in the eXist-db git repo. 
+* the list of dependency updates can be prepared with
     ```
-    project.version = 3.1.0-SNAPSHOT
+    gitchangelog eXist-5.3.0..HEAD | grep "Updates" | sort > dependency-updates.txt
     ```
+    Replace **eXist-5.3.0** with the tag of the latest release.
 
-2.  Commit the changes and push to `origin` (or `upstream` if you are on a fork).
 
 ### Preparing a Product Release
 
-Once development on a new stable version is complete, the following steps will prepare the version for release. For purposes of illustration, we will assume we are preparing the stable release of version 5.3.0.
-You will require a system with:
-* macOS
-* JDK 8
-* Maven 3.6.0+
-* Docker
-* GnuPG
-* A GPG key (for signing release artifacts)
-* A Java KeyStore with key (for signing IzPack Installer)
-* A valid Apple Developer Certificate (for signing Mac DMG)
-* A Github account and username / password or Github Personal access tokens (https://github.com/settings/tokens) with permission to publish Github releases to the eXist-db .org
-
-1. You will need login credentials for the eXist-db organisation on:
-    1. Sonatype OSS staging for Maven Central - https://oss.sonatype.org/
-    2. DockerHub - https://cloud.docker.com/orgs/existdb/
+#### Maven settings.xml
+
+Required credentials and the Maven `existdb-release-signing` profile should be stored securely in the `<servers>` section on your machine in your local `~/.m2/settings.xml` file. The contents should follow the example below.
```

Comment:
This needs rewording - the `existdb-release-signing` profile does NOT go in the `<servers>` section